instrumentation-tests/src/androidTest/java/com/mapbox/navigation/instrumentation_tests/core/RouteRefreshTest.kt

```diff
@@ -329,8 +377,33 @@ class RouteRefreshTest : BaseTest<EmptyTestActivity>(EmptyTestActivity::class.ja
             alternativesMetadata.navigationRoute.id
         )
         assertNotEquals(alternativesMetadataLegacy, alternativesMetadata)
-        assertEquals(227.9, alternativesMetadataLegacy.infoFromStartOfPrimary.duration, 1.0)
-        assertEquals(235.0, alternativesMetadata.infoFromStartOfPrimary.duration, 1.0)
+        assertEquals(227.918, alternativesMetadataLegacy.infoFromStartOfPrimary.duration, 0.001)
+        // fork index is 4. So take 4 durations from primary refresh response
```

Comment:
@RingerJK I divided the test in 2. Here both routes are refreshed, in the test above - only the primary one. And I added the comments on how to calculate the expected values. 235 is a valid expected value for the case where only the primary route is refreshed (that's what was happening before but on the SDK side, not because of the server).